src/isyntax/isyntax.c

```diff
@@ -2606,7 +2611,7 @@ void isyntax_set_work_queue(isyntax_t* isyntax, work_queue_t* work_queue) {
 	isyntax->work_submission_queue = work_queue;
 }
 
-bool isyntax_open(isyntax_t* isyntax, const char* filename) {
+bool isyntax_open(isyntax_t* isyntax, const char* filename, bool init_allocators) {
```

Comment:
note the api change - I want to be able to suppress default per-isyntax allocators, as they now can be shared. Not perfect API, but works.

Reply:
Makes sense. Maybe we can refine it later.